hw/ip/padctrl/data/padctrl_fpv_testplan.hjson

```diff
@@ -0,0 +1,167 @@
+{
+  name: "padctrl"
+  import_testplans: ["hw/dv/tools/testplans/fpv_csr_testplan.hjson"]
+  entries: [
+    {
+      name: MioWarl_A
+      desc: '''In either generic or Xilinx mode, if a muxed IO's pad attribute is written through
+            the TLUL interface via the mio_pads registers, this assertion checks if the
+            corresponding mio_attr_o's reserved bits remain 0. This assertion intends to test the
+            right_any_read_legal access policy for these mio_pads registers.'''
+      milestone: V2
+      tests: ["padctrl_assert"]
+    }
+    {
+      name: MioAttr_A
+      desc: '''In either generic or Xilinx mode, if a muxed IO's pad attribute is written through
+            the TLUL interface via mio_pads registers, this assertion checks if the corresponding
+            mio_attr_o value is updated correctly.'''
+      milestone: V2
+      tests: ["padctrl_assert"]
+    }
+    {
+      name: MioBackwardCheck_A
+      desc: '''If the output mio_attr_o has changed, then in the previous clock cycle, there must be a
```

Comment:
I would probably reformulate this as:

If the output mio_attr_o has changed, then a valid write to the mio_pads register must have taken place or the write enable signal must have risen in the previous cycle.

Same comment for `DioBackwardCheck_A` below.